catalog/core/catalog-core-impl/pubsub/src/main/java/ddf/catalog/pubsub/EventProcessorImpl.java

```diff
@@ -435,8 +436,48 @@ public DeleteResponse process(DeleteResponse deleteResponse) throws PluginExecut
         return deleteResponse;
     }
 
-    public static enum DateType {
-        modified, effective, expiration, created
+    /**
+     * Enumeration of metacard Date attributes that can be used for subscriptions. In order to use
+     * metacard attribute names for getting DateType values, DateType.getDateType(metacardAttr)
```

Comment:
✏️  You can use the `{@link className#methodName}` syntax to refer to a method name in JavaDoc comments. It creates an hyperlink in the HTML and helps the IDEs update the comments when method names change. `className` can be omitted if the method is in the current class, and `className` by itself can be used when referring to the whole class.